server/src/main/java/io/crate/metadata/CoordinatorTxnCtx.java

```diff
@@ -37,7 +37,7 @@
 public final class CoordinatorTxnCtx implements TransactionContext {
 
     private final SessionContext sessionContext;
-    private Long currentTimeMillis = null;
+    private volatile Instant currentInstant;
```

Comment:
Before `currentTimeMillis` wasn't volatile, so it shouldn't be necessary for `currentInstant` to be volatile now.